.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
You're an owner on the github operationzombie organization, so you've got control over all the repos in there.

I've manually added you as admin for those two repos, so hopefully it should be sorted.
(You might need to get travis to re-sync your repos to recognize new permissions though)